Source/WebCore/Modules/mediastream/UserMediaRequest.cpp

```diff
@@ -139,7 +139,8 @@ void UserMediaRequest::start()
         break;
     }
 
-    PlatformMediaSessionManager::singleton().prepareToSendUserMediaPermissionRequest();
+    if (RefPtr page = document.protectedPage())
+        PlatformMediaSessionManager::singleton().prepareToSendUserMediaPermissionRequestForPage(*page);
```

Comment:
This is probably safer this way but normally, we should be guaranteed to have a page here.
Maybe ASSERT(document.page()) to document this.